pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -10,18 +9,127 @@
 from .. import debug as debug
 from .. import getConfigOption
 
-__all__ = ['PlotDataItem']
+__all__ = ['PlotDataItem', 'PlotDataset']
```

Comment:
If `PlotDataset` is not used outside of this file, we should probably remove it from `__all__` (as `__all__` handles the case where you input `*`)

Reply:
This was primarily added to make sphinx find it when coming from the TOC list.
And without the TOC list entry, a sphinx warning ("no TOC entry") failed the CI tests.
--> Flagging the PlotDataset docstring as `:orphan:` seems to avoid this.